pkgs/development/libraries/spice/default.nix

```diff
@@ -39,10 +39,10 @@ stdenv.mkDerivation rec {
 
 
   nativeBuildInputs = [
+    glib
```

Comment:
Do we need glib in buidlInputs? I was able to remove it and the package still built

Reply:
It might be propagated by something, but it looks needed:
https://gitlab.freedesktop.org/spice/spice/-/blob/master/server/event-loop.c#L128

(I hope it's the right repo)